docs/getting_started/examples.md

```diff
@@ -1,83 +1,77 @@
+- [Adding exporters](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/cont/adding_exporters/) - Add new exporters to mbed tools.
+- [Port new mbed platform](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/porting_guide/) - Port a new mbed platform.
+- [Migrating from mbed OS 3 - MINAR](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/MINAR_migration/) - Port mbed OS 3 to mbed OS 5.
+- [mbed BLE](https://docs.mbed.com/docs/ble-intros/en/latest/) - Use an mbed Enabled Bluetooth Low Energy (BLE) platform to build. [Note: Contains 7 parts.]
+- [Battery level](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-BatteryLevel/) - Create a battery level service containing a single GATT characteristic using Bluetooth Low Energy (BLE).
+- [BLE beacon](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-Beacon/) - Create a Bluetooth Low Energy (BLE) beacon to send information to a nearby device.
+- [BLE button](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-Button/) - See an example about a Bluetooth Low Energy (BLE) service template.
+- [GAP button](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-GAPButton/) - Use GAP to transmit a value to disconnected peer listening when a value updates using Bluetooth Low Energy (BLE).
+- [BLE heart rate monitor](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-HeartRate/) - Transmit a heart rate using the Bluetooth SIG Heart Rate Profile.
+- [BLE LED control](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-LED/) - Control an LED through a phone app using Bluetooth Low Energy (BLE).
+- [BLE thermometer](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-Thermometer/) - Send thermometer information using Bluetooth Low Energy (BLE).
+- [Eddystone beacon and web BLE](https://github.com/armmbed/mbed-os) - Use the Eddystone beacon with Bluetooth Low Energy (BLE). Try it with the [Eddystone observer](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-EddystoneObserver/), which scans for beacons.
+- [Building your own private LoRa network](https://docs.mbed.com/docs/lora-with-mbed/en/latest/intro-to-lora/) - Set up a LoRA gateway and endpoints.
+- [Connected lighting](https://docs.mbed.com/docs/building-an-internet-connected-lighting-system/en/latest/) - Build an internet connected lighting system. [Note: Contains 8 parts]
+- [Compile time configuration](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/config_system/) - Guide to the mbed configuration system.
+- [mbed TLS hardware acceleration](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/tls_hardware_acceleration/) - Add and use hardware acceleration to mbed TLS. [Note: Duplicated in TLS.]
```

Comment:
Ordered by usefulness / likelyhood of developer to care. 

TLS has its own community and its own examples, cross listing form here is a low priority item. Majority of users will encounter TLS as a built in part of mbed-os, not as a seperate entity. Same for uvisor (which I honestly dont think should even be surfaced as its not a released product, the uvisor team is not willing to support issues, so all we're doing is creating confusion and frustration)